docs/source/quickstart.rst

```diff
@@ -1,9 +1,49 @@
 Quickstart
 ==========
 
-Run the main code via:
-```
-python jfsd/JFSD.py
-```
+## How to Run Simulations
 
-During the simulation, the particles trajectories,velocities, and stresslets are saved in a numpy array of shape (N_s, N_p, N_c), with N_s the number of frames stored, N_p the number of particles and N_c the number of d.o.f. (3 for trajectories, 6 for velocities, 5 for stresslets).
+1 - Create a configuration .toml in files/ (use "files/example_configuration.toml" as reference)
```

Comment:
I would leave out that it needs to be created in files/, it can be anywhere the researcher wants.